### ISSUE TITLE
vulnerability issue

### ISSUE THREAD

**Issue by aquiladev** [sc-forks/solidity-coverage#227]
npm threw it to me after audit:

```
                                              Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Critical        Command Injection

  Package         growl

  Patched in      >=1.10.2

  Dependency of   solidity-coverage [dev]

  Path            solidity-coverage > solidity-parser-sc > mocha > growl

  More info       https://nodesecurity.io/advisories/146


  High            Regular Expression Denial of Service

  Package         minimatch

  Patched in      >=3.0.2

  Dependency of   solidity-coverage [dev]

  Path            solidity-coverage > solidity-parser-sc > mocha > glob >
                  minimatch

  More info       https://nodesecurity.io/advisories/118


  Low             Regular Expression Denial of Service

  Package         debug

  Patched in      >= 2.6.9 < 3.0.0 || >= 3.1.0

  Dependency of   solidity-coverage [dev]

  Path            solidity-coverage > solidity-parser-sc > mocha > debug

  More info       https://nodesecurity.io/advisories/534
```

**Comment by cgewecke** [comment]
@aquiladev Thanks! Will see if I can bump `mocha` in the parser to get rid of this warning. Slightly mystified by `audit` though because warning is about a dev dep of a dev dep of a dev dep. Shouldn't be installing anywhere except at solidity-parser-sc AFAIK.

**Comment by cgewecke** [comment]
Should be fixed in `0.5.1`. Turns out mocha is not a dev dep over there.

**Comment by cgewecke** [comment]
@aquiladev Apologies, messed this up yesterday - didn't move mocha up high enough. It should actually be fixed now.
